packages/flutter/test/material/about_test.dart

```diff
@@ -1841,6 +1841,147 @@ void main() {
       expect(renderParagraph.text.style!.color, theme.primaryTextTheme.titleLarge!.color);
     });
   });
+
+  testWidgets('Adaptive AboutDialog shows correct widget on each platform',(WidgetTester tester) async {
+    for (final TargetPlatform platform in <TargetPlatform>[ TargetPlatform.iOS, TargetPlatform.macOS]) {
```

Comment:
Small nit: The contents of this `for` block are indented by 4 spaces instead of 2 as per the style guide.